src/Http/Routing/src/Builder/RoutingEndpointConventionBuilderExtensions.cs

```diff
@@ -22,6 +22,15 @@ public static class RoutingEndpointConventionBuilderExtensions
     /// An empty collection means any host will be accepted.
     /// </param>
     /// <returns>A reference to this instance after the operation has completed.</returns>
+    /// <remarks>
+    /// API that relies on the <see href="https://developer.mozilla.org/docs/Web/HTTP/Headers/Host">Host header</see>, such as
```

Comment:
Plural here will match the plural "are subject to" in the next line.

```suggestion
    /// APIs that rely on the <see href="https://developer.mozilla.org/docs/Web/HTTP/Headers/Host">Host header</see>, such as
```